docs/development.rst

```diff
@@ -1355,6 +1355,21 @@ Once all this is done, submit a PR containing the code changes and wait for dire
 on whom to send the compressed archive of genetic maps to (currently Andrew Kern is the
 primary uploader but please wait to send files to him until directed).
 
+**An important note:**
+when an existing resource file (such as a genetic map or annotation)
+is updated and replaces the previous version,
+be sure to update the name of the file pattern with a version number
+(i.e. ``name_{id}_more_name_v1.tar.gz``).
```

Comment:
This seems confused - either `{id}` should not go here (because as above it refers to an individual chromosome, and we *don't* rename the individual chromosome files?) or else this is the wrong thing that needs to be changed (ie we don't change the `file_pattern`)?

Reply:
Yeah, I think this is wrong. Note [below in your PR](https://github.com/popsim-consortium/stdpopsim/pull/1553/files#diff-7525483765c11f21a5dd836cee64d70bcb66e5f70f2c52246be45d6e649c4989L17); you are not changing `file_pattern`; you're changing like `intervals_url`. Here you should be documenting/explaining the very process you're doing in this PR.

Reply:
Yes sorry, thanks for looking carefully!